projects/plugins/jetpack/tests/e2e/lib/env/playwright-environment.js

```diff
@@ -210,7 +210,7 @@ class PlaywrightCustomEnvironment extends NodeEnvironment {
 		await this.saveScreenshot( eventFullName );
 		await this.logHTML( eventFullName );
 
-		if ( E2E_DEBUG && PAUSE_ON_FAILURE && this.global.page ) {
+		if ( ( E2E_DEBUG || PAUSE_ON_FAILURE ) && this.global.page ) {
```

Comment:
this change is intentional. When E2E_DEBUG is used, the execution is paused right away, plus the output is too verbose